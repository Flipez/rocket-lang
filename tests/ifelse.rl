i = 0
while (i<2)
  if (i==0)
    puts("if/else: if block")
  else
    puts("if/else: else block")
  end

  i = i+1
end
