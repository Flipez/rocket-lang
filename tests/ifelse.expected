"if/else: if block"
"if/else: else block"
nil
